docs/doxygen.css

```diff
@@ -313,8 +313,8 @@ HR { height: 1px;
 
 /* 
  * LLVM Modifications.
- * Note: Everything above here is generated with "doxygen -w htlm" command. See
  * "doxygen --help" for details. What follows are CSS overrides for LLVM 
+ * Note: Everything above here is generated with "doxygen -w html" command. See
```

Comment:
Oh, damn sorry. That happened because my editor removes all trailing whitespace which most of sources have here, so it is a bit difficult to stage changes, as there is a lot of "noise" from whitespace removal